flow/duration.yaml

```diff
@@ -104,6 +104,12 @@ components:
         A continuous burst of packets that will not automatically stop.
       type: object      
       properties:
+        burst_count:
+          description: >-
+            The number of burst transmitted per stream.
```

Comment:
- please rephrase description to remove mention of stream. e.g. "The number of packet bursts transmitted per flow. A value of 0 implies continuous burst of packets."
- we should call it `bursts` (instead of burst_count) to be inline with `packets`

Reply:
Done.